cmd/botkube/main.go

```diff
@@ -20,85 +20,117 @@
 package main
 
 import (
-	"fmt"
 	"os"
 
+	"github.com/infracloudio/botkube/pkg/filterengine"
+
+	"golang.org/x/sync/errgroup"
+
 	"github.com/infracloudio/botkube/pkg/bot"
 	"github.com/infracloudio/botkube/pkg/config"
 	"github.com/infracloudio/botkube/pkg/controller"
 	"github.com/infracloudio/botkube/pkg/log"
-	"github.com/infracloudio/botkube/pkg/metrics"
 	"github.com/infracloudio/botkube/pkg/notify"
 	"github.com/infracloudio/botkube/pkg/utils"
+
+	"github.com/infracloudio/botkube/pkg/metrics"
```

Comment:
The imports can be divided into 2 parts 
```
<3rd-party import>

<local packages>
```
Please group the local packages remove the unnecessary new lines. 
```

Reply:
I ran goimports again, now it should be fine 👍